doc/internal-api/local.mk

```diff
@@ -0,0 +1,7 @@
+$(docdir)/internal-api/html/index.html $(docdir)/internal-api/latex: $(d)/doxygen.cfg
+	mkdir -p $(docdir)/internal-api
+	{ cat $< ; echo "OUTPUT_DIRECTORY=$(docdir)/internal-api" ; } | doxygen -
```

Comment:
I noticed that this is also generating LaTeX docs in `$doc/share/doc/nix/internal-api/latex`, can that be disabled?

Reply:
Sure